.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,36 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+<!--This is just a template - feel free to delete any and all of it and replace as appropriate.-->
+
+**Description**
```

Comment:
Nit: perhaps use a header here?

Reply:
Could you clarify -- what would it look like? I do want to minimize the words in new issues that are common to all issues, because it makes them less efficient to read. I think our goal is only to improve the quality of the issue reports.

Reply:
```suggestion
# Description
```

Reply:
Done -- `####` gives about the right size.